src/common/registry/storage/operator.entity.ts

```diff
@@ -39,4 +41,8 @@ export class RegistryOperator {
 
   @Property()
   usedSigningKeys!: number;
+
+  @PrimaryKey()
```

Comment:
The same comment as for srmodule. I think it's not a primary key. It should have a unique constraint instead.